packages/nc-gui/components/template/Editor.vue

```diff
@@ -615,6 +617,13 @@ function handleEditableTnChange(idx: number) {
 function isSelectDisabled(uidt: string, disableSelect = false) {
   return (uidt === UITypes.SingleSelect || uidt === UITypes.MultiSelect) && disableSelect
 }
+
+function handleCheckAllRecord(event, table_name) {
```

Comment:
parameter implicitly has an 'any' type - set it when you know the type

```ts
import type { CheckboxChangeEvent } from 'ant-design-vue/es/checkbox/interface'
```

```ts
function handleCheckAllRecord(event: CheckboxChangeEvent, table_name: string) {
```

Reply:
thankyou for your suggest.
I have changed the code